cpp/CMakeLists.txt

```diff
@@ -15,13 +15,15 @@ option(MEMILIO_SANITIZE_ADDRESS "Enable address sanitizer." OFF)
 option(MEMILIO_SANITIZE_UNDEFINED "Enable undefined behavior sanitizer." OFF)
 option(MEMILIO_BUILD_SHARED_LIBS "Build memilio as a shared library." ON)
 option(MEMILIO_BUILD_STATIC_LIBS "Build memilio as a static library." ON)
+option(MEMILIO_USE_BUNDLED_IPOPT "Use bundled Ipopt, requires a fortran compiler" ON)
 option(MEMILIO_ENABLE_MPI "Build memilio with MPI." OFF)
 option(MEMILIO_ENABLE_OPENMP "Enable Multithreading with OpenMP." OFF)
 
 mark_as_advanced(MEMILIO_USE_BUNDLED_SPDLOG MEMILIO_SANITIZE_ADDRESS MEMILIO_SANITIZE_UNDEFINED)
 
 set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake" ${CMAKE_MODULE_PATH})
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+set(CMAKE_OSX_DEPLOYMENT_TARGET 14) # avoid linker warnings (built for newer OSX version than being linked)

```

Comment:
necessary? if?

Reply:
It is note really necessary. It just avoid spurious linker warnings on MacOS. However, it can be deleted.

Reply:
you can set it for OSX in an "if" with the given comment

Reply:
Now, I check for APPLE.